### PR TITLE
Typo correction in README.org of the bepo layer.

### PR DESCRIPTION
**Pull request by benquebec**
Correction of typos and changed wording in a few places in the documentation of the bepo layer.

### CODE REVIEW

File: layers/+keyboard-layouts/bepo/README.org [hunk 1]
```diff
@@ -43,13 +43,13 @@ used for *doing a movement*:
   - ~s → k~
   - ~r → l~
 
-- Map lost functionnalites back to some keys:
+- Map lost functionalites back to some keys:
```
Comment (StreakyCobra) [inline layers/+keyboard-layouts/bepo/README.org]: You missed on missing `i` here I think

File: layers/+keyboard-layouts/bepo/README.org [hunk 2]
```diff
@@ -170,7 +169,7 @@ keys being shown after a small delay following a keypress.
 |-------------+-------------------------------+--------------------|
 | Key Binding | Description                   | Replace/equivalent |
 |-------------+-------------------------------+--------------------|
-| ~k~         | Evil-surround functionnalites | ~s~                |
+| ~k~         | Evil-surround functionalites | ~s~                |
```
Comment (StreakyCobra) [inline layers/+keyboard-layouts/bepo/README.org]: Try to keep the table's lines aligned when you do changes :-)

**Comment by StreakyCobra** [comment]
You have a new opportunity to improve your `squash`-fu :smile:

PS: In this case, it's also possible to apply the new changes, amend the commit you already did with these new changes, and then force the push. I let you do what you prefer :wink:

File: layers/+keyboard-layouts/bepo/README.org (hunk 2 above)
Comment (benquebec) [inline layers/+keyboard-layouts/bepo/README.org]: oups, forgot to put an "i"

**Comment by StreakyCobra** [comment]
@benquebec `Merge branch …` Probably not what you wanted to do :smiley: You can try again the procedure I gave you on gitter to squash the commits.

**Comment by benquebec** [comment]
I was told to do that after git commit --amend. ;(

**Comment by TheBB** [comment]
Thank you! Cherry-picked in develop. You can safely delete your branch.

**Pull request closed without merging**